tests/release/test_fixture_args.py

```diff
@@ -0,0 +1,96 @@
+import os
+import subprocess
+from pathlib import Path
+
+import pytest
+
+from pseudopeople.interface import (
+    generate_american_community_survey,
+    generate_current_population_survey,
+    generate_decennial_census,
+    generate_social_security,
+    generate_taxes_1040,
+    generate_taxes_w2_and_1099,
+    generate_women_infants_and_children,
+)
+from tests.release.conftest import (
+    DEFAULT_ENGINE,
+    DEFAULT_STATE,
+    DEFAULT_YEAR,
+    FULL_USA_FILEPATH,
+    RI_FILEPATH,
+    _parse_dataset_params,
```

Comment:
shouldn't be private if used outside of the conftest

Reply:
Isn't everything declared in a conftest file automatically available to all test files in its package? Or is that only fixtures?

Reply:
They're available if I include them in the args to test functions but I can't seem to access them outside of those which is what I need to define my dictionary.